content/auth/identified-clients.textile

```diff
@@ -48,17 +48,13 @@ For example, when publishing a message, the @clientId@ attribute of the message
 The following example demonstrates how to issue an "Ably TokenRequest":/auth/token#token-request with an explicit @clientId@:
 
 ```[realtime_javascript]
-  var realtime = new Ably.Realtime({ key: '{{API_KEY}}' });
-  realtime.auth.createTokenRequest({ clientId: 'Bob' }, function(err, tokenRequest) {
-    /* ... issue the TokenRequest to a client ... */
-  })
+  const realtime = new Ably.Realtime.Promise({ key: '{{API_KEY}}' });
+  const requestToken = realtime.auth.createTokenRequest({ clientId: 'Bob'});
 ```
 
 ```[realtime_nodejs]
-  var realtime = new Ably.Realtime({ key: '{{API_KEY}}' });
-  realtime.auth.createTokenRequest({ clientId: 'Bob' }, function(err, tokenRequest) {
-    /* ... issue the TokenRequest to a client ... */
-  })
+  const realtime = new Ably.Realtime.Promise({ key: '{{API_KEY}}' });
+  const requestToken = realtime.auth.createTokenRequest({ clientId: 'Bob'});
```

Comment:
```suggestion
  const tokenRequest = await realtime.auth.createTokenRequest({ clientId: 'Bob'});
```

Reply:
(I'll stop adding comments for these but there are more examples of this below)

Reply:
https://github.com/ably/docs/pull/2119/commits/e90046ac431bd699719987715d1edab1ac28e59d

I found a couple other references.